appinventor/components/src/com/google/appinventor/components/runtime/util/TextViewUtil.java

```diff
@@ -168,13 +168,8 @@ public static Typeface getTypeFace(Form form, String fontFile) {
     Typeface typeface;
     if (!fontFile.contains("/")) {
       if (form instanceof ReplForm) {
-        if (Build.VERSION.SDK_INT > 28) {
-          File file = new File("/storage/emulated/0/Android/data/edu.mit.appinventor.aicompanion3/files/assets/" + fontFile);
-          typeface = Typeface.createFromFile(file);
-        } else {
-          File file = new File("/storage/emulated/0/AppInventor/assets/" + fontFile);
-          typeface = Typeface.createFromFile(file);
-        }
+        File file = new File("/storage/emulated/0/Android/data/edu.mit.appinventor.aicompanion3/files/assets/" + fontFile);
```

Comment:
It would probably be better to make MediaUtil.fileUrlToFilePath public and then do

```java
File file = new File(MediaUtil.fileUrlToFilePath(form.getAssetPath(fontFile)));
```

Since getAssetPath is also used to write the asset during the connection step, this would ensure that any future changes to where the assets are written are respected. By hard-coding the path there may be future breakage if we decide to change the location of assets (or have it forced on us as was the case previously with the file scope changes).

Reply:
I made changes.